### DISCUSSION TITLE
Unable to install package

### DISCUSSION THREAD

**Discussion by naro-code-lab** [spatie/laravel-backup#1463] [Q&A] [answered]
I encountered this error while installing package....

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/laravel-backup[8.0.0, ..., 8.0.1] require illuminate/console ^9.0 -> found illuminate/console[v9.0.0-beta.1, v9.0.0-beta.2, v9.0.0-beta.3, 9.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires spatie/laravel-backup ^8.0 -> satisfiable by spatie/laravel-backup[8.0.0, 8.0.1].

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

I  currently have laravel 8 and php 8 installed.

**Comment by brandon-9** [comment]
Same issue here. L8, PHP8. It does work with version 7.7.3

**Comment by erikn69** [comment, marked as answer]
`spatie/laravel-backup:^8.0` only works with `laravel:^9.0`,
for `laravel:^8.0` use `spatie/laravel-backup:^7.0`
